### PR TITLE
2nd Edition + package.json cleanup

### PR DESCRIPTION
**Pull request by gillesgros**
Seaside2, Hinterlands2, Proisperity2, Package.json updated to last version except vue.js

### CODE REVIEW

**Comment by gillesgros** [comment]
Hello,

I initialted a global pull request [#139 2nd Edition + package.json cleanup](https://github.com/blakevanlan/KingdomCreator/pull/139)](https://github.com/blakevanlan/KingdomCreator/pull/139)

I am not feeling confident in my knowledge of Git.
I will feel much more at ease if you review and do the merge.
  I need to practice on my side.

Regards,
Gilles

File: webpack.dev.js [hunk 1]
```diff
@@ -1,11 +1,22 @@
-const merge = require("webpack-merge");
+/* webpack 5 */
+const { merge } = require('webpack-merge');
+/* webpack 3
```
Comment (markbrockettrobson) [inline webpack.dev.js:3]: do we want to get rid of all the old code ?

File: plugins/dominion-content-plugin.js [hunk 2]
```diff
@@ -102,7 +102,9 @@ module.exports = class DominionContentPlugin {
     for (let i = 0; i < files.length; i++) {
       const filename = path.join(directory, files[i]);
       const id = this.tokenize(path.basename(files[i], '.yaml'));
-      values[id] = yaml.safeLoad(fs.readFileSync(filename, 'utf8'));
+	  /* js-yamj v3 to v4
```
Comment (markbrockettrobson) [inline plugins/dominion-content-plugin.js:105]: do we want to remove the old v3 code ?

File: process/loader.js [hunk 3]
```diff
@@ -40,7 +40,10 @@ const loadFilesFromDirectory = function(directory) {
    for (let i = 0; i < files.length; i++) {
       const filename = path.join(directory, files[i]);
       const id = tokenize(path.basename(files[i], '.yaml'));
-      values[id] = yaml.safeLoad(fs.readFileSync(filename, 'utf8'));
+      /* js-yaml  v3 to v4
```
Comment (markbrockettrobson) [inline process/loader.js:43]: old code

**Review by markbrockettrobson — CHANGES_REQUESTED** [review]
We need to add some changes from 115 :

 src/randomizer/randomizer.ts:
         some changes to make Colonies show up from PROSPERITY and PROSPERITY_2
kingdoms/nocturne.yaml:
         tag the kingdom "Day at the Races" with       
         - nocturne_boon_theswampsgift
         - nocturne_boon_theriversgift
         - nocturne_boon_theforestsgift

**Comment by gillesgros** [comment]
Changes made.
 Thanks for the review

**Comment by markbrockettrobson** [comment]
We just need the changes to 
randomizer to make Colonies show up from PROSPERITY and PROSPERITY_2

You will find the in the 115pr

**Comment by gillesgros** [comment]
Fix + update of kngdoms for proper colonies use. same for shelter

**Review by markbrockettrobson — APPROVED** [review]

**Pull request merged**